src/__tests__/components/articleBody.spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ArticleBody from '../../components/articleBody/ArticleBody';
+
+describe('Article body shallow tests', () => {
```

Comment:
@Dubby20 a better `describe` name would be "ArticleBody component" since that's what we're testing.

Reply:
I've changed the test to be more descriptive